src/summary/documentation/vitals-card.test.tsx

```diff
@@ -0,0 +1,99 @@
+import React from "react";
+import { mockPatient } from "../../../__mocks__/patient.mock";
+import { cleanup, render, wait } from "@testing-library/react";
+import { BrowserRouter } from "react-router-dom";
+import VitalsCard from "./vitals-card.component";
+import { of } from "rxjs/internal/observable/of";
+import * as openmrsApi from "./vitals-card.resource";
+
+describe("<VitalsCard/>", () => {
+  let patient: fhir.Patient, match;
+  const mockVitalsResponse = [
+    {
+      id: 1463379216000,
+      date: "2016-05-16T06:13:36.000+00:00",
+      systolic: 161,
+      diastolic: 72,
+      pulse: 22,
+      temperature: 37,
+      oxygenation: 30
+    },
+    {
+      id: 144048423500012,
+      date: "2015-08-25T06:30:35.000+00:00",
+      systolic: 156,
+      diastolic: 64,
+      pulse: 173,
+      temperature: 37,
+      oxygenation: 41
+    }
+  ];
+  afterEach(() => {
+    cleanup;
+  });
+
+  beforeEach(() => {
+    patient = mockPatient;
+    match = { params: {}, isExact: false, path: "/", url: "/" };
+  });
+
+  it("renders without dying", () => {
+    const wrapper = render(
+      <BrowserRouter>
+        <VitalsCard match={match} patient={patient} />
+      </BrowserRouter>
+    );
+    expect(wrapper).toBeDefined();
+  });
+
+  it("should display the patients vitals correctly", async () => {
+    const spy = jest
+      .spyOn(openmrsApi, "performPatientsVitalsSearch")
```

Comment:
When you call `spyOn`, you have to call `spy.mockRestore` after you're done with it. Even easier though is just to call [mockRestoreAll](https://jestjs.io/docs/en/jest-object#jestrestoreallmocks) inside of an `afterEach()` block

Reply:
Made the requested changes and added vitals.mock.ts file that contains mocked network response.